test/unit/app/services/asset_manager/attachment_updater_test.rb

```diff
@@ -208,7 +173,7 @@ class AssetManager::AttachmentUpdaterTest < ActiveSupport::TestCase
 
           AssetManager::AssetUpdater.expects(:call).never
 
-          AssetManager::AttachmentUpdater.call(attachment.attachment_data, redirect_url: true, draft_status: true)
+          AssetManager::AttachmentUpdater.call(attachment.attachment_data)
```

Comment:
Should this also be calling `AssetManager::AttachmentUpdater.redirect`? The old test behaviour seems to suggest it but I'm not 100% sure

Reply:
So that test was making a call that the application never does. There was nowhere in the app that passed those two flags at the same time, so it didn't seem worth having a test case that covered both